crates/lox-io/src/ndm/ocm.rs

```diff
@@ -0,0 +1,1156 @@
+                    "This example shows the specification of multiple comment lines".to_string(),
+                ],
+                classification_list: vec![],
+                creation_date: common::EpochType(
+                    "1998-11-06T09:23:57".to_string(),
+                ),
+                originator: "JAXA".to_string(),
+                message_id: Some(
+                    "OCM 201113719185".to_string(),
+                ),
+            },
+            body: OcmBody {
+                segment: OcmSegment {
+                    metadata: OcmMetadata {
+                        comment_list: vec![],
+                        object_name: None,
+                        international_designator: Some(
+                            "1998-999A".to_string(),
+                        ),
+                        catalog_name: None,
+                        object_designator: None,
+                        alternate_names: None,
+                        originator_poc: Some(
+                            "R. Rabbit".to_string(),
+                        ),
+                        originator_position: Some(
+                            "Flight Dynamics Mission Design Lead".to_string(),
+                        ),
+                        originator_phone: Some(
+                            "(719)555-1234".to_string(),
+                        ),
+                        originator_email: None,
+                        originator_address: None,
+                        tech_org: None,
+                        tech_poc: Some(
+                            "Mr. Rodgers".to_string(),
+                        ),
+                        tech_position: None,
+                        tech_phone: Some(
+                            "(719)555-1234".to_string(),
+                        ),
+                        tech_email: None,
+                        tech_address: Some(
+                            "email@email.XXX".to_string(),
+                        ),
+                        previous_message_id: None,
+                        next_message_id: None,
+                        adm_msg_link: None,
+                        cdm_msg_link: None,
+                        prm_msg_link: None,
+                        rdm_msg_link: None,
+                        tdm_msg_link: None,
+                        operator: None,
+                        owner: None,
+                        country: None,
+                        constellation: None,
+                        object_type: None,
+                        time_system: "UT1".to_string(),
+                        epoch_tzero: common::EpochType(
+                            "1998-12-18T00:00:00.0000".to_string(),
+                        ),
+                        ops_status: None,
+                        orbit_category: None,
+                        ocm_data_elements: None,
+                        sclk_offset_at_epoch: None,
```

Comment:
Do you think there would be value in exercising both the `None` and `Some` cases for each of the fields?

Reply:
Yeah, that makes sense to me. Carry on!